### ISSUE TITLE
Show blocked by bruteforce

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#893]
Users are sometimes confused why the login times out.
This is often due to bruteforce blocking/delaying the login to 30s.

How can we get a better user experience for this?

**Comment by AndyScherzinger** [comment]
Yes, please! This might prevent some recurring questions coming in via the forum, tracker, etc. due to the lack of given information by the app.

**Comment by AndyScherzinger** [comment]
Would help with #433 for example

**Comment by tobiasKaminsky** [comment]
As a small brainstorm with Lukas:
- https://github.com/nextcloud/server/issues/4510 will implement an endpoint to see how long the password timeout is
- if > 10s then show a small "warning: timeout is due to BF adjusted. Please wait" (or similar) and then adjust the timeout to 1.5 mins or such.

**Comment by mario** [comment]
Sure, this was discussed at the last hack week.

**Comment by tobiasKaminsky** [comment]
> Sure, this was discussed at the last hack week.

Then an issue with such an info and roadmap would have been nice to not redo the work ;-)

**Comment by mario** [comment]
I would have expected my friend @LukasReschke has done it already :P

**Comment by tobiasKaminsky** [comment]
At least not here in android repo ;-)

**Comment by LukasReschke** [comment]
Evil evil Lukas 😉

**Comment by tobiasKaminsky** [comment]
As the real solution via a proper server timeout will take some time (NC 13, https://github.com/nextcloud/server/issues/4510#issuecomment-301591234) I suggest to just enhance the timeout to > 30s, probably 45s. This way all "blocked" users will connect again and the other have to wait just a little bit longer to get the "connection timed out" message.
@mario @AndyScherzinger

**Comment by AndyScherzinger** [comment]
Sounds good to me 👍

**Comment by joshtrichards** [comment]
Fixed in nextcloud/android-library#65
The balance is a duplicate of #1178
